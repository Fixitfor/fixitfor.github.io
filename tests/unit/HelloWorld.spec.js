import Vue from 'vue'
import Vuetify from 'vuetify'
Vue.use(Vuetify);

import HelloWorld from '@/components/HelloWorld'

import {
  mount,
  createLocalVue
} from '@vue/test-utils'

const localVue = createLocalVue();

describe('HelloWorld.vue', () => {
  let vuetify;

  beforeEach(() => {
    vuetify = new Vuetify()
  });

  it('should match snapshot', () => {
    const wrapper = mount(HelloWorld, {
      localVue,
      vuetify,
    });
    // With jest we can create snapshot files of the HTML output
    expect(wrapper.html()).toMatchSnapshot()
  })
});