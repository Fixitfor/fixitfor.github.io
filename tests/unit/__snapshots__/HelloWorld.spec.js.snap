// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue should match snapshot 1`] = `<div class="container amazing-app"></div>`;
